addon/components/frost-object-browser.js

```diff
@@ -22,6 +22,9 @@ export default Ember.Component.extend({
   classNames: ['frost-object-browser'],
   contentHeight: 505,
   detailLevel: 'low',
+  isHighDetailVisible: true,
+  isLowDetailVisible: true,
+  isMediumDetailVisible: false,
```

Comment:
I worry about adding additional configuration options when there isn't a need. If we already have all the information necessary to support the use-cases, I'd prefer not to add additional API surface area.